### PR TITLE
Hcal online dev

### PR DESCRIPTION
**Pull request by mrsolt**
Updating the HCal online monitoring.
- Remove unnecessary/redundant file (mostly monitors)
- updates to HCal monitoring
- updates to event display  
- adding Hcal daq map and Hcal gains file
- add parsing script to daq and gains file (much improved from october)
- adding sample config and setup scripts

### CODE REVIEW

File: software/eudaq/include/eudaq/CSVParser.h [hunk 1]
```diff
@@ -0,0 +1,178 @@
+#ifndef EUDAQ_CSVPARSER_H
+#define EUDAQ_CSVPARSER_H
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <iterator>
+#include <boost/tokenizer.hpp>
```
Comment (tomeichlersmith) [inline software/eudaq/include/eudaq/CSVParser.h:10]: Is Boost installed on the CERN computers?

**Review by tomeichlersmith — COMMENTED** [review]
Besides the Boost comment, the only work left is updating to my redesign of the decoding. This decoding is already on `main` and I have updated this branch with that code.

The new decoding functions like
```cpp
auto data = hcal::decode(event.GetBlock(1));
for (const auto& [el, samples] : data) {
  // el is hcal::ElectronicsLocation for this readout channel
   // samples is std::vector<hcal::HgcrocSample> for this readout channel
}
```
both `ElectronicsLocation` and `HgcrocSample` are defined in `include/eudaq/HgcrocRawDecode.h` header.

File: software/eudaq/include/eudaq/CSVParser.h (hunk 1 above)
Comment (mrsolt) [reply software/eudaq/include/eudaq/CSVParser.h:10]: According to PF, boost is on the CERN computers.

**Comment by omar-moreno** [comment]
I'm merging this in to get main working without disabling monitoring.  I need this so I can start setting up TS monitoring.

**Pull request merged**